File: .ProgramingBasics/SimpleConditionalStatements/09.PasswordGuess/src/PasswordGuess.java
import java.util.Scanner;

public class PasswordGuess {
    public static void main(String[] args) {
        final String password = "s3cr3t!P@ssw0rd";
        Scanner scanner = new Scanner(System.in);
        String input = scanner.nextLine();

        if (input.equals(password)) {
            System.out.println("Welcome");
        }
        else {
            System.out.println("Wrong password!");
        }
    }
}

File: ProgramingBasics/SimpleConditionalStatements/09.PasswordGuess/09.PasswordGuess/PasswordGuess.cs
using System;

namespace _09.PasswordGuess
{
    public class PasswordGuess
    {
        public static void Main(string[] args)
        {
            const string password = "s3cr3t!P@ssw0rd";
            var input = Console.ReadLine();

            if (input == password)
            {
                Console.WriteLine("Welcome");
            }
            else
            {
                Console.WriteLine("Wrong password!");
            }
        }
    }
}
